src/main/kotlin/de/nofelix/abyssgazer/role/stat/CharStatManager.kt

```diff
@@ -261,4 +275,41 @@ class CharacterStats(private val provider: PlayerEntity) : CharStat, AutoSyncedC
     private fun isCurrentStatAtOrBelowMilestone(stat: Int): Boolean {
         return stat <= STAT_MILESTONE
     }
+
+    /**
+     * Calculates the required experience level for the next level up.
+     * Returns different XP requirements based on the current player level:
+     * - Levels 1-15: current level + 5
+     * - Levels 16-30: current level + 3
+     * - Levels 31-50: current level + 2
+     * - Levels 51-75: current level + 1
+     * - Levels 76+: current level (no increase)
+     *
+     * Example:
+     * - At level 5: requires 5 + 5 = 10 XP-level to level up
+     * - At level 25: requires 25 + 3 = 28 XP-level to level up
+     * - At level 40: requires 40 + 2 = 42 XP-level to level up
+     * - At level 60: requires 60 + 1 = 61 XP-level to level up
+     * - At level 100: requires 100 + 0 = 100 XP-level to level up
+     *
+     * @return The experience level required to level up
+     */
+    private fun calculateLevelUpCondition(): Int {
+        return when {
+            this.level <= 15 -> this.level.plus(EXP_BASE_LEVEL)
+            this.level <= 30 -> this.level.plus(EXP_BASE_LEVEL.minus(2))
+            this.level <= 50 -> this.level.plus(EXP_BASE_LEVEL.minus(3))
+            this.level <= 75 -> this.level.plus(EXP_BASE_LEVEL.minus(4))
+            else -> this.level
+        }
+    }
+
+    private fun receiveLevelUpPoints() {
+        this.levelUpPoints = LVL_POINTS_VALUE
```

Comment:
warum nicht addieren?
Verliert man nicht dann seine alten Punkte bei nem Levelup?
```suggestion
        this.levelUpPoints += LVL_POINTS_VALUE
```